### DISCUSSION TITLE
Obico Server in LXC

### DISCUSSION THREAD

**Discussion by sjfm-design** [tteck/Proxmox#2641] [Ideas]
hi!
ich versuche seit 3 tagen sowas aufzusetzen. anscheinend zu dämlich.
https://www.obico.io/docs/server-guides/
das mit docker schaff ich einfach nicht.
siehst du da eine möglichkeit?

**Comment by tteck** [comment]
English only.

**Reply by sjfm-design** [reply]
k.
tried since 3 days to install an obico server in a proxmox lxc. maybe to stupid.
oh, there is a script for docker. i'll try this...
but maybe a script for obico?
